pop/service/api.go

```diff
@@ -213,6 +225,20 @@ func encodeMapFinal(stmts map[string]*FinalStatement) ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
+// ToToml returns a string that can be decoded using toml.
+func (desc *PopDesc) ToToml() (string, error) {
```

Comment:
where is this function used? further, this ToToml isn't consistent with FinalStatement.ToToml, which returns ([]byte, error).

Reply:
Removed it - it's a mess:
- reading from the file it's a `PopDescGroupToml`
- getting from the service it's a `PopDesc`
- `shortDescToml` is used for the final statement

So probably `shortDescToml` should be replaced by `PopDescGroupToml`. That's an exercise for later...